contexts/PaymentContext/tests/Unit/Domain/ChecksumGeneratorTest.php

```diff
@@ -43,6 +43,19 @@ public function testCanGenerateChecksumWithManyCharacters(): void {
 		$this->assertSame( '9', $generator->createChecksum( 'AAAAXXXX-' ) );
 	}
 
+	public function testLowerAndMixedCaseProduceConsistentChecksum(): void {
+		$generator = new ChecksumGenerator( str_split( 'ACDEFKLMNPRSTWXYZ349' ) );
+
+		$this->assertSame( 'F', $generator->createChecksum( 'aAaa-aaaa-' ) );
+		$this->assertSame( 'F', $generator->createChecksum( 'aaaa-aaaa-' ) );
+		$this->assertSame( '4', $generator->createChecksum( 'abcd-efGh-' ) );
+		$this->assertSame( '4', $generator->createChecksum( 'abcd-efgh-' ) );
+		$this->assertSame( 'K', $generator->createChecksum( 'QaQa-qaqa-' ) );
+		$this->assertSame( 'K', $generator->createChecksum( 'qaqa-qaqa-' ) );
+		$this->assertSame( '9', $generator->createChecksum( 'aaAa-XXXX-' ) );
+		$this->assertSame( '9', $generator->createChecksum( 'aaaaxxxx-' ) );
```

Comment:
sneaky refugee test hiding in this method :)

Reply:
Well, I warned that the master would be red with this.

Reply:
I mean that this last line is also testing something else